src/number_theory/primorial.lean

```diff
@@ -94,16 +95,16 @@ lemma primorial_le_4_pow : ∀ (n : ℕ), n# ≤ 4 ^ n
   | or.inl n_odd :=
     match nat.even_iff.2 n_odd with
     | ⟨m, twice_m⟩ :=
-      let recurse : m + 1 < n + 2 := by linarith in
+      have recurse : m + 1 < n + 2 := by linarith,
       begin
         calc (n + 2)#
             = ∏ i in filter nat.prime (range (2 * m + 2)), i : by simpa [←twice_m]
         ... = ∏ i in filter nat.prime (finset.Ico (m + 2) (2 * m + 2) ∪ range (m + 2)), i :
               begin
                 rw [range_eq_Ico, finset.union_comm, finset.Ico_union_Ico_eq_Ico],
-                exact bot_le,
-                simp only [add_le_add_iff_right],
-                linarith,
+                { exact bot_le },
+                { simp only [add_le_add_iff_right, two_mul],
+                  apply nat.le_add_left },
```

Comment:
```suggestion
                { simpa only [add_le_add_iff_right, two_mul] using nat.le_add_left },
```
if you like (maybe there might be some needed underscores after the `nat.le_add_left`